File: src/main/java/no/nav/pensjon/regler/domain/grunnlag/Pensjonsbeholdning.java
package no.nav.pensjon.regler.domain.grunnlag;

import no.nav.pensjon.regler.domain.kode.BeholdningsTypeCti;

public class Pensjonsbeholdning extends Beholdning {
    private static final long serialVersionUID = 7664743814937303739L;

    public Pensjonsbeholdning() {
        super();
        beholdningsType = new BeholdningsTypeCti("PEN_B");
    }

    public Pensjonsbeholdning(Pensjonsbeholdning pb) {
        super(pb);
    }
}

File: src/main/kotlin/no/nav/pensjon/regler/domain/grunnlag/Pensjonsbeholdning.kt
package no.nav.pensjon.regler.domain.grunnlag

import no.nav.pensjon.regler.domain.enum.BeholdningtypeEnum

open class Pensjonsbeholdning : Beholdning() {
    override var beholdningsTypeEnum: BeholdningtypeEnum = BeholdningtypeEnum.PEN_B
}